examples/neptune_tracking/README.md

```diff
@@ -0,0 +1,145 @@
+# 🏋️ Track your runs with Neptune
+
+[Neptune](https://neptune.ai/product/experiment-tracking) is a
+popular platform that you would normally use in the iterative ML experimentation
+phase to track and visualize experiment results. Neptune can also
+track and visualize the results produced by your
+automated pipeline runs, as you make the transition towards a more production
+oriented workflow.
+
+## 🗺 Overview
+This example showcases how to use the Neptune Experiment Tracker flavor to log and visualize information from your pipeline steps (e.g. models, parameters, metrics).
+
+We'll be using the [MNIST](http://yann.lecun.com/exdb/mnist/) dataset and
+will train a classifier using [Tensorflow (Keras)](https://www.tensorflow.org/).
+We will run two experiments with different parameters (epochs and learning rate)
+and log these experiments into a neptune platform. 
+
+In the example script, the [neptune-tensorflow-keras](https://docs.neptune.ai/integrations/keras/) is
+used within the training step to directly hook into the TensorFlow training.  
+It will log out all relevant parameters, metrics and output files Automatically. . Additionally, we explicitly log the test accuracy within the evaluation step.
+
+Note that despite `neptune` being used in different steps within a pipeline, ZenML handles initializing `neptune` and ensures the experiment name is the same as the pipeline name, and the experiment run is the same name as the pipeline run name. This establishes a lineage between pipelines in ZenML and experiments in `neptune`.
+
+# 🖥 Run it locally
```

Comment:
I think in other examples we also include the "super quick run" using `zenml example run`, I think we should add that here as well

Reply:
Wouldn't super quick run need stack component configuration to connect to their platform?

Reply:
Ah you're absolutely right, Neptune only has the remote server right? In that case we should delete the `setup.sh` file in order to raise the correct warning/error when trying `zenml example run`